index.html

```diff
@@ -24,22 +25,6 @@
                title="Bazel"
                alt="Bazel logo"
                class="logo img-responsive" />
-          <p class="tagline">{Fast, Correct} - Choose two</p>
```

Comment:
We use to have simply "<> Bazel" as the big thing, and we can use the same, have the tag line under the logo and where the logo is have the "Build and test software of any size, quickly and reliabily" in big, with the button underneath?